.github/workflows/release.yaml

```diff
@@ -121,11 +108,22 @@ jobs:
         # Verify hashes by first computing hashes for the artifacts and
         # then comparing them against the hashes computed by the build job.
         echo "$ARTIFACT_HASH" | base64 -d | sha256sum --strict --check --status || exit 1
+    # Create the Release Notes using commitizen.
+    - name: Set up Python
+      uses: actions/setup-python@b55428b1882923874294fa556849718a1d7f2ca5 # v4.2.0
+      with:
+        python-version: '3.10'
+    - name: Set up Commitizen
+      run: |
+        pip install --upgrade pip wheel
+        pip install 'commitizen ==2.32.1'
+    - name: Create Release Notes
+      run: cz changelog --dry-run $(cz version --project) > RELEASE_NOTES.md
```

Comment:
Nice, this is the same as before?
```
cz changelog --dry-run --incremental --unreleased-version "Release Notes" > RELEASE_NOTES.md
```

Reply:
> Nice, this is the same as before?
> 
> ```
> cz changelog --dry-run --incremental --unreleased-version "Release Notes" > RELEASE_NOTES.md
> ```

Same output as before but we shouldn't run `--incremental` at this point because there are no new commits.

Reply:
… which in turn addressed https://github.com/jenstroeger/python-package-template/issues/301 👍🏼